lib/transactions/vote.js

```diff
@@ -1,6 +1,20 @@
-var crypto = require("./crypto.js"),
-    constants = require("../constants.js"),
-    slots = require("../time/slots.js");
+/**
+ * Crypto module helping with creating votes in Lisk Blockchain
```

Comment:
Vote module provides functions for creating vote transactions for use with the Lisk Blockchain.